spec/station12/reservations_controller_spec.rb

```diff
@@ -1,112 +1,66 @@
 require 'rails_helper'
 
-RSpec.describe Admin::ReservationsController, type: :controller do
+RSpec.describe ReservationsController, type: :controller do
   render_views
-  describe 'Station12 GET /admin/reservations' do
-    before do
-      sheets = create_list(:sheet, 3)
-      schedule = create(:schedule, movie_id: create(:movie).id)
-      @first_reservation = create(:reservation, schedule_id: schedule.id, sheet_id: sheets.first.id)
-      create(:reservation, schedule_id: schedule.id, sheet_id: sheets[1].id) # 予約を3つ作成する
-      @last_reservation = create(:reservation, schedule_id: schedule.id, sheet_id: sheets.last.id)
-      get :index
+  describe 'Station12 GET /movies/:movie_id/schedules/:schedule_id/reservations/new' do
+    let(:movie) { create(:movie) }
+    let(:sheets) { create_list(:sheet, 5) }
+    let(:date) { "2023-01-31" }
+    let(:schedule) { create(:schedule, movie_id: movie.id) }
+    let(:success_request) { get :new, params: { movie_id: movie.id, schedule_id: schedule.id, date: date, sheet_id: sheets.first.id }, session: {} }
+    let(:no_date_and_sheet_request) { get :new, params: { movie_id: movie.id, schedule_id: schedule.id }, session: {} }
+    let(:no_date_request) { get :new, params: { movie_id: movie.id, schedule_id: schedule.id, sheet_id: sheets.first.id }, session: {} }
+    let(:no_sheet_request) { get :new, params: { movie_id: movie.id, schedule_id: schedule.id, date: date }, session: {} }
+
+    context "クエリについて" do
```

Comment:
```suggestion
    context 'クエリについて' do
```

ここだけダブルクオートの意味もないので、シングルクオートで良いかも？